import os
import sys

import numpy as np
import pandas as pd

from src.exception import CustomException

def load_object(file_path):
    try:
        with open(file_path, "rb") as file_obj:
            return dill.load(file_obj)

    except Exception as e:
        raise CustomException(e, sys)